### ISSUE TITLE
tables not displaying properly

### ISSUE THREAD

**Issue by bnicenboim** [cognitive-science-stan/cognitive-science-stan.github.io#16]
Maybe use https://glin.github.io/reactable/articles/examples.html#searching

**Comment by bnicenboim** [comment]
https://www.johnmackintosh.net/blog/2022-01-05-reactable/

**Comment by bnicenboim** [comment]
The tables are now usable, but I think they could be much wider taking the window width as width. And since one needs to scroll down, I guess showing all the entries (or maybe a very large number) should be the default.

**Comment by cuierd** [comment]
Just to make sure I understand you clearly: you mean make the table wider so that it will show all the columns by default (or most of the columns). Under this case, the users do not need to scroll right/left; but they still need to scroll down in order to see more entries (rows), for example, when they choose to display 100 entries.

**Comment by bnicenboim** [comment]
yes, I think all the columns won't fit, but even if they fit they will need to scroll right/left if they happen to open the page in a small window. All these elements react to the width of the displayed window so that it's usually taken care by default.
